packages/frontend/src/components/MenuBar/NavigationDropdownSubMenuProfile.tsx

```diff
@@ -73,7 +73,7 @@ export const NavigationDropdownSubMenuProfile: React.FC<DropdownSubMenuProps> =
           fullWidth
         />
         {filteredParties.map((party) => {
-          const companyName = party.partyType === 'Organization' ? toTitleCase(party.name || '') : '';
+          const companyName = toTitleCase(party.name || '');
```

Comment:
change to `partyName` e..g.? Maybe fallback value should be something else, too. Just to make it clear ...